knowledge-base/comments-vs-notes-in-radspreadprocessing.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Comments vs Notes in RadSpreadProcessing
+description: Learn how to set comments and notes on cells in RadSpreadProcessing for Document Processing and understand the difference between Comments and Notes in Excel.
+type: how-to
+page_title: Comments vs Notes in RadSpreadProcessing
+slug: comments-vs-notes-in-radspreadprocessing
+tags: comments, notes, excel, spreadprocessing
+res_type: kb
+---
+# Comments vs Notes in RadSpreadProcessing
+
+## Environment
+
+| Version | Product | Author | 
+| --- | --- | ---- | 
+| 2024.1.124 | RadWordsProcessing |[Desislava Yordanova](https://www.telerik.com/blogs/author/desislava-yordanova)| 
+
+## Description
+When opening XLSX files in MS Excel, the comments may appear as threaded comments. This article explains the reason behind this behavior.
+
+![Threaded Comments in MS Excel](images/comments-vs-notes-in-radspreadprocessing001.png) 
+
+## Solution
+
+The [SpreadProcessing]({%slug radspreadprocessing-overview%}) library supports both [Comments]({%slug radspreadprocessing-features-comments%}) and [Notes]({%slug radspreadprocessing-features-notes%}) in Excel. The difference between them is as follows:
+
+- **Comments**: These are the traditional comments that were available in earlier versions of Excel. They are displayed as small red triangles in the corners of cells and can be viewed by hovering over the cell. Each comment can be replied to, forming a thread of information. 
```

Comment:
The notes are related to "red triangles" but the comments to purple (some kind of) shapes.